traffic_ops/traffic_ops_golang/deliveryservice/consistenthash/consistenthash.go

```diff
@@ -0,0 +1,117 @@
+package consistenthash
+
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+import (
+	"database/sql"
+	"encoding/json"
+	"errors"
+	"github.com/apache/trafficcontrol/traffic_ops/traffic_ops_golang/api"
+	"io/ioutil"
+	"net/http"
+	"net/url"
+	"strconv"
+)
+
+type TRConsistentHashResult struct {
```

Comment:
This package and the types within it could use some comments describing what they are and what they do. Could be as simple as naming the endpoint that a `func` serves, or the concept that a `type` represents. All Go code hosted on GitHub.com is documented using [godoc](https://blog.golang.org/godoc-documenting-go-code), so we should try to make it helpful (there are also some limited formatting options you can read about in the docs)